src/components/dls/Button/Button.tsx

```diff
@@ -59,6 +60,7 @@ const Button: React.FC<ButtonProps> = ({
   variant,
   tooltip,
   className,
+  noSidePadding = false,
```

Comment:
I think the name is a bit confusing. Having the negation would be easier e.g. `hasSidePadding` which by default = `true` would be better